lib/templates/_footer.html

```diff
@@ -26,6 +26,7 @@
 <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
 <script src="static-assets/typeahead.bundle.min.js"></script>
 <script src="static-assets/prettify.js"></script>
+<script src="index.js"></script>
```

Comment:
We may also want to consider an `async` or a `defer` here so that loading the script doesn't block page rendering.